packages/kbn-es/src/cli_commands/serverless.ts

```diff
@@ -66,20 +70,28 @@ export const serverless: Command = {
     });
     const reportTime = getTimeReporter(log, 'scripts/es serverless');
 
-    const argv = process.argv.slice(2);
+    // replacing --serverless with --projectType when 'scripts/es --serverless=<projectType>' is called
+    const argv = process.argv.slice(2).map((arg) => arg.replace('serverless', 'projectType'));
```

Comment:
I was thinking about a better way to do it. We know that `argv[0]` is either `serverless` or `--serverless=<>` and can use it override the first arg in case of the later option.
```
    const argv = process.argv.slice(2);
    if (argv[0].startsWith('--serverless')) {
      const projectTypeArg = argv[0].replace('--serverless', '--projectType');
      argv[0] = projectTypeArg;
    }
```